build/media-suite_compile.sh

```diff
@@ -1992,30 +1992,30 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
 
     _check=(libvulkan.a vulkan.pc)
     if ! mpv_disabled vulkan &&
-        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git#tag=v1.1.114" vulkan-loader; then
-        _shinchiro_patches="https://raw.githubusercontent.com/shinchiro/mpv-winbuild-cmake/b1f8bc13/packages"
+        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git" vulkan-loader; then
+        _DeadSix27="https://raw.githubusercontent.com/DeadSix27/python_cross_compile_script/master"
         do_uninstall "${_check[@]}"
-        do_patch "$_shinchiro_patches/vulkan-0001-cross-compile-static-linking-hacks.patch"
+        do_patch "$_DeadSix27/patches/vulkan/0001-fix-cross-compiling.patch"
         create_build_dir
         log dependencies /usr/bin/python3 ../scripts/update_deps.py --no-build
         cd_safe Vulkan-Headers
-            _check=(vulkan/vulkan.h)
-            do_uninstall include/vulkan
+            _check=(vulkan/vulkan.h d3d{kmthk,ukmdt}.h)
+            do_uninstall include/vulkan d3d{kmthk,ukmdt}.h
```

Comment:
No need to keep uninstalling them, these types of headers don't get updated that often.